### PR TITLE
[ticket/16083] Remove common W3C validation errors and warnings

### PR DESCRIPTION
**Pull request by senky**
Checklist:

- [x] Correct branch: master for new features; 3.3.x & 3.2.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html), [3.3.x](https://area51.phpbb.com/docs/dev/3.3.x/development/coding_guidelines.html) and [3.2.x](https://area51.phpbb.com/docs/dev/3.2.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.3.x/development/git.html)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-16083

### CODE REVIEW

File: phpBB/styles/prosilver/template/navbar_header.html [hunk 1]
```diff
@@ -183,14 +183,14 @@
...
 
 			<!-- BEGIN navlinks -->
 				<!-- EVENT overall_header_navlink_prepend -->
-				<span class="crumb" {$MICRODATA}<!-- IF navlinks.MICRODATA --> {navlinks.MICRODATA}<!-- ENDIF -->><a href="{navlinks.U_VIEW_FORUM}" itemtype="https://schema.org/Thing" itemprop="item"><span itemprop="name">{navlinks.FORUM_NAME}</span></a><meta itemprop="position" content="{{ navlink_position }}{% set navlink_position = navlink_position + 1 %}" /></span>
+				<span class="crumb" {$MICRODATA}<!-- IF navlinks.MICRODATA --> {navlinks.MICRODATA}<!-- ENDIF -->><a href="{navlinks.U_VIEW_FORUM}" itemtype="https://schema.org/Thing"itemscope itemprop="item"><span itemprop="name">{navlinks.FORUM_NAME}</span></a><meta itemprop="position" content="{{ navlink_position }}{% set navlink_position = navlink_position + 1 %}" /></span>
```
Comment (dark-1) [inline phpBB/styles/prosilver/template/navbar_header.html]: Need to add space :
```
itemtype="https://schema.org/Thing" itemscope
                              here ^
```

Comment (dark-1) [inline phpBB/styles/prosilver/template/navbar_header.html]: Need to add space :
```
itemtype="https://schema.org/Thing" itemscope
                              here ^
```

**Comment by dark-1** [comment]
Thanks @senky ,

I think few issues with `Google Search Engine` with regards to `Breadcrumbs` will be resolved...

Best regards 👍

File: phpBB/adm/style/acp_attachments.html [hunk 2]
```diff
@@ -110,7 +110,7 @@ <h2>{L_UPLOADING_FILES}</h2>
 <!-- ELSEIF S_EXTENSION_GROUPS -->
 
 	<!-- IF S_EDIT_GROUP -->
-		<script type="text/javascript" defer="defer">
+		<script defer="defer">
```
Comment (mrgoldy) [inline phpBB/adm/style/acp_attachments.html]: As per https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#attr-defer
Having the `defer="defer"` for an in-line script without a `src=""` is pointless.

**Pull request merged**